### PR TITLE
Pw 6688 checkout api v69

### PR DESCRIPTION
**Pull request by wboereboom**
**Description**
Updated Checkout API models to v69

**Tested scenarios**
updated and ran all unit tests
ran e2e tests

### CODE REVIEW

File: src/main/java/com/adyen/model/checkout/PaymentRefundResource.java [hunk 1]
```diff
@@ -33,33 +33,31 @@
 import java.util.List;
 import com.adyen.model.Amount;
 import com.adyen.model.Split;
+import io.swagger.v3.oas.annotations.media.Schema;
```
Comment (AlexandrosMor) [inline src/main/java/com/adyen/model/checkout/PaymentRefundResource.java:36]:
```suggestion
```

File: src/main/java/com/adyen/model/checkout/Phone.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+ *
+ * Copyright (c) 2022 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+
+package com.adyen.model.checkout;
+
+import com.google.gson.annotations.SerializedName;
+import io.swagger.v3.oas.annotations.media.Schema;
```
Comment (AlexandrosMor) [inline src/main/java/com/adyen/model/checkout/Phone.java:25]:
```suggestion
```

File: src/main/java/com/adyen/model/checkout/SDKEphemPubKey.java [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+ * Copyright (c) 2022 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ */
+
+package com.adyen.model.checkout;
+
+import com.google.gson.annotations.SerializedName;
+import io.swagger.v3.oas.annotations.media.Schema;
```
Comment (AlexandrosMor) [inline src/main/java/com/adyen/model/checkout/SDKEphemPubKey.java:26]:
```suggestion
```

**Review by michaelpaul — APPROVED** [review]

**Comment by michaelpaul** [comment]
@wboereboom I noticed that we don't have a `CardDetails` model yet. It's used on `/payments` https://github.com/Adyen/adyen-openapi/blob/main/json/CheckoutService-v69.json#L8780. Could you add it too in this PR?

**Comment by wboereboom** [comment]
@michaelpaul in the Java lib this is taken care of by `DefaultPaymentMethodDetails`.

**Review by michaelpaul — APPROVED** [review]

**Review by AlexandrosMor — APPROVED** [review]

**Pull request merged**